civis/parallel.py

```diff
@@ -761,18 +775,24 @@ class _CivisBackend(ParallelBackendBase):
 
     Users should interact with this through ``make_backend_factory``.
     """
+    uses_threads = False
+    supports_sharedmem = False
+    supports_timeout = True
+
     def __init__(self, setup_cmd=_DEFAULT_SETUP_CMD,
                  from_template_id=None,
                  max_submit_retries=0,
                  client=None,
                  remote_backend='sequential',
+                 nesting_level=0,
                  **executor_kwargs):
         self.setup_cmd = setup_cmd
         self.from_template_id = from_template_id
         self.max_submit_retries = max_submit_retries
         self.client = client
         self.remote_backend = remote_backend
         self.executor_kwargs = executor_kwargs
+        self.nesting_level = nesting_level
```

Comment:
What does it mean for this to be `>0`?

Reply:
The `nesting_level` attribute is to handle the case where you get a `Parallel` call inside a `Parallel` call, e.g. a multiprocessing Estimator called inside a cross-validation loop. I added it because it's now part of the `ParallelBackendBase` signature (https://github.com/joblib/joblib/blob/acca6b95ed9e28498441d630ed2ddf674b66c8b8/joblib/_parallel_backends.py#L36). This function looks at it: https://github.com/joblib/joblib/blob/acca6b95ed9e28498441d630ed2ddf674b66c8b8/joblib/_parallel_backends.py#L120 . We're not using it, so for now the purpose is just making sure we accept all of the parameters that the parent class does.